packages/nc-gui/components/smartsheet/toolbar/QrScannerButton.vue

```diff
@@ -0,0 +1,162 @@
+<script setup lang="ts">
+import type { SelectProps } from 'ant-design-vue'
+import { ref } from 'vue'
+import { StreamBarcodeReader } from 'vue-barcode-reader'
+import type { ColumnType } from 'nocodb-sdk'
+import { UITypes } from 'nocodb-sdk'
+import { NOCO } from '#imports'
+import QrCodeScan from '~icons/mdi/qrcode-scan'
+
+const meta = inject(MetaInj, ref())
+
+const { t } = useI18n()
+const route = useRoute()
+const router = useRouter()
+
+const { $api } = useNuxtApp()
+const { project } = useProject()
```

Comment:
since we've introduced pinia, this line should be

```vue
const { project } = storeToRefs(useProject())
```

you can import `storeToRefs`  from `'#imports`